workflow/rocoto/gfs_tasks.py

```diff
@@ -665,6 +665,33 @@ def prepoceanobs(self):
 
         return task
 
+    def marineanalletkf(self):
+
+        ocean_hist_path = self._template_to_rocoto_cycstring(self._base["COM_OCEAN_HISTORY_TMPL"], {'RUN': 'gdas'})
+
+        deps = []
+        data = f'{ocean_hist_path}/gdas.ocean.t@Hz.inst.f009.nc'
```

Comment:
will that build the dependency on all ensemble members?

Reply:
No, perhaps we have to rethink the task depencies

Reply:
done